dnf/cli/commands/repolist.py

```diff
@@ -71,7 +71,7 @@ class RepoListCommand(commands.Command):
     repolist command.
     """
 
-    aliases = ('repolist', 'repoinfo')
+    aliases = ('repolist', 'repoinfo', 'rlist',)
```

Comment:
``repolist`` and ``rlist`` is nearly the same, `rls` is not nice. What about to not provide anything here.